decred/tests/unit/dcr/test_txscript.py

```diff
@@ -4333,3 +4333,84 @@ def test_multi_sig_script():
         assert (
             res == test["want"]
         ), f'wanted {test["want"].hex()} but got {res.hex()} for test {test["name"]}'
+
+
+def test_pay_to_addr_script():
+    psf = parseShortForm
+    hash160 = crypto.hash160(0)
+    addrPubKeyHash = addrlib.AddressPubKeyHash(
+        hash160, testnet, crypto.STEcdsaSecp256k1
+    )
+    addrScriptHash = addrlib.AddressScriptHash(hash160, testnet)
+    addrPubKeyHashEdwards = addrlib.AddressPubKeyHash(
+        hash160, testnet, crypto.STEcdsaSecp256k1
+    )
+    addrPubKeyHashEdwards.sigType = crypto.STEd25519
+    addrPubKeyHashSchnorr = addrlib.AddressPubKeyHash(
+        hash160, testnet, crypto.STEcdsaSecp256k1
+    )
+    addrPubKeyHashSchnorr.sigType = crypto.STSchnorrSecp256k1
+    pubBytes = psf("0x03 {}".format(hex(Curve.curve.N)))
+    addrSecpPubKey = addrlib.AddressSecpPubKey(pubBytes, testnet)
+    addrEdwardsPubKey = addrlib.AddressEdwardsPubKey.__new__(
+        addrlib.AddressEdwardsPubKey
+    )
+    addrSchnorrPubKey = addrlib.AddressSecSchnorrPubKey.__new__(
+        addrlib.AddressSecSchnorrPubKey
+    )
+    """
+    name (str): Short description of the test.
+    addr (Address): An address.
+    wantException (Exception): The exception expected if any.
+    want (ByteArray): The pay to address script.
+    """
+    tests = [
+        dict(
+            name="address pubkey hash",
+            addr=addrPubKeyHash,
+            want=psf(
+                "DUP HASH160 0x{} EQUALVERIFY CHECKSIG".format(
+                    txscript.addData(hash160).hex()
+                )
+            ),
+        ),
+        dict(
+            name="edwards hash not implemented",
+            addr=addrPubKeyHashEdwards,
+            wantException=NotImplementedError,
+        ),
+        dict(
+            name="schnorr hash not implemented",
+            addr=addrPubKeyHashEdwards,
```

Comment:
This should be `addrPubKeyHashSchnorr`.

Reply:
I could've sworn I had already fixed this.